### PR TITLE
[CTOR-495] Update Gude EPC SNMP

### PR DESCRIPTION
**Pull request by YanMeddour**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] Cloud
- [ ] Monitoring Connectors

### CODE REVIEW

**Review by omercier — DISMISSED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-pdu-gude-epc-snmp.md [hunk 1]
```diff
@@ -72,88 +98,367 @@ Le Pack Gude Export Power Control collecte les données pour:
 
 Afin de contrôler votre Gude Export Power Control, le SNMP doit être configuré.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-pdu-gude-epc-snmp.md:99]: ?

File: pp/integrations/plugin-packs/procedures/hardware-pdu-gude-epc-snmp.md [hunk 2]
```diff
@@ -73,89 +98,367 @@ The Pack Gude Export Power Control collects metrics for:
 To monitor your Gude Export Power Control, the SNMP must be configured.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/hardware-pdu-gude-epc-snmp.md:98]: ?

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-pdu-gude-epc-snmp.md (hunk 1 above)
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-pdu-gude-epc-snmp.md:99]:
```suggestion
### Configuration SNMP

Le service SNMP doit être activé et configuré sur l'équipement. Veuillez vous référer à la documentation officielle du constructeur/éditeur.

### Flux réseau

La communication doit être possible sur le port UDP 161 depuis le collecteur
Centreon vers la ressource supervisée.
```

File: pp/integrations/plugin-packs/procedures/hardware-pdu-gude-epc-snmp.md (hunk 2 above)
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/hardware-pdu-gude-epc-snmp.md:98]:
```suggestion
### SNMP Configuration

The SNMP service must be configured and activated on the host. Please refer to the official documentation from the manufacturer/publisher.

### Network flow

The target resource must be reachable from the Centreon poller on the UDP/161
SNMP port.
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by cg-tw — APPROVED** [review]

**Review by omercier — APPROVED** [review]

**Pull request merged**